lib/modules/datasource/deb/readme.md

```diff
@@ -34,6 +36,7 @@ First you would set a custom manager in your `renovate.json` file for `Dockerfil
       "matchStrings": [
         "#\\s*renovate:\\s*?depName=(?<depName>.*?)?\\sENV .*?_VERSION=\"(?<currentValue>.*)\"\\s"
       ],
+      "registryUrlTemplate": "https://deb.debian.org/debian?suite=stable&components=main,contrib,non-free&binaryArch=amd64",
```

Comment:
In the implementation MR, @viceice told remove it since this is the default.

Reply:
I have changed the example so it makes actual use of the template part of `registryUrlTemplate`